v2/pkg/reporting/trackers/jira/jira.go

```diff
@@ -109,12 +112,21 @@ func jiraFormatDescription(event *output.ResultEvent) string {
 	builder.WriteString("\n\n*Timestamp*: ")
 	builder.WriteString(event.Timestamp.Format("Mon Jan 2 15:04:05 -0700 MST 2006"))
 	builder.WriteString("\n\n*Template Information*\n\n| Key | Value |\n")
-	for k, v := range event.Info {
-		if k == "reference" {
+
+	fields := reflect.TypeOf(event.Info)
```

Comment:
These very few lines of code enable reusability and refactor-safe retrieval of all field names and their values from a specified struct. Since it is only being used after the scan, at the reporting stage, the performance "penalty" is **negligible**. Manually running a benchmark test over and over again, or printing the results on the screen produces very similar execution times, while running them in a loop without printing, the manual string concatenation approach is 3x times faster, which means a difference of `0.00165` MILLIseconds! 

The current result using reflection look something similar to:
```
| Name | model.Info struct field name and value retrieval benchmark |
| Authors | {[forgedhallpass ice3man]} |
| Tags | {[fuzz dos cve]} |
| Description | Some very descriptive text |
| Reference | {[reference1 reference2 reference3]} |
| SeverityHolder | {high} |
```
Since the output could be better looking, I can accept going with a manual version, that would look something like:
```
| Name | model.Info struct field name and value retrieval benchmark |
| Authors | forgedhallpass, ice3man |
| Tags | forgedhallpass, ice3man |
| Description | Some very descriptive text |
| Severity | high |
```